src/util/emitUtils.js

```diff
@@ -549,6 +549,22 @@ function getNewObjAsString(key, obj) {
   return JSON.stringify(wrappedObj);
 }
 
+/**
+ * In the case of multiple connections to the same socket.io room
+ * (ie. multiple instances of the same bot) this function will emit
+ * data to only one.
+ * @param {Socket.io} io - socket.io server
+ * @param {String} nsp - name of namespace
+ * @param {String} room - id of room within namespace
+ * @param {Socket.io} namespace - object representing namespace
+ */
+function emitToSingleInstance(io, namespace, namespaceId, room) {
```

Comment:
@ConalCosgrove  The order and names of the function params don't match the ones in function comment. Can you double check? 
Also, can you give an example of what the 'namespace' would be? (Let's say for On-call Bot)

Reply:
good catch, thanks! had to update to include the namespaceId.